### PR TITLE
feat: run autoprefixer on css

### PR DESCRIPTION
**Pull request by BrandonOCasey**
## Description
`autoprefixer` cuts out any vendor prefixes that we don't need. It would probably be better to run this on the source files, but we cannot since we are working with sass. So for now we can run it as a "minification" step. 

## Results
* New size: `9514` bytes gziped
* Old size: `9829` bytes gziped

### CODE REVIEW

**Comment by gkatsev** [comment]
We should try and not add stuff to grunt. We probably should add an npm script and then use grunt shell to call it.

File: build/grunt.js [hunk 1]
```diff
@@ -398,9 +406,10 @@ module.exports = function(grunt) {
     'shell:lint',
     'clean:build',
 
-    'shell:rollupall',
+    // 'shell:rollupall',
```
Comment (gkatsev) [inline build/grunt.js]: this probably shouldn't be commented out.

Comment (BrandonOCasey) [reply build/grunt.js]: woops yeah, I was testing with it commented out.

**Comment by gkatsev** [comment]
This probably doesn't need to be ported to 6.x.

**Pull request merged**